src/Core/Scripting/OOJSPlayerShip.m

```diff
@@ -724,6 +724,15 @@ static JSBool PlayerShipSetProperty(JSContext *context, JSObject *this, jsid pro
 			}
 			break;
 
+		case kPlayerShip_primedEquipment:
+			sValue = OOStringFromJSValue(context, *value);
+			if (sValue != nil)
+			{
+				[player setPrimedEquipment:sValue showMessage:YES];
+				return YES;
```

Comment:
I think we should be able to return also NO here, in case of failure to set primed equipment. Since -setPrimedEquipment: returns BOOL anyway, I would simply go with
```
return [player setPrimedEquipment:sValue showMessage:YES];
```